src/Mod/PartDesign/Gui/TaskHoleParameters.cpp

```diff
@@ -792,259 +792,145 @@ void TaskHoleParameters::changedObject(const App::Document&, const App::Property
 {
     auto hole = getObject<PartDesign::Hole>();
     if (!hole) {
-        // happens when aborting the command
-        return;
+        return; // happens when aborting the command
     }
     bool ro = Prop.isReadOnly();
 
     Base::Console().Log("Parameter %s was updated\n", Prop.getName());
 
+    auto updateCheckable = [&](QCheckBox* widget, bool value) {
+        QSignalBlocker blocker(widget);

```

Comment:
I bet some compilers complain about the "unused" variable here, you might consider annotating with `[[maybe_unused]]`.